repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py

```diff
@@ -191,7 +191,7 @@ def _get_lsblk_info():
 @aslist
 def _get_pvs_info():
     """ Collect storage info from pvs command """
-    for entry in _get_cmd_output(['pvs', '--noheadings', '--separator', r':'], ':', 6):
+    for entry in _get_cmd_output(['pvs', '--noheadings', '--separator', r'|'], ':', 6):
```

Comment:
```suggestion
    for entry in _get_cmd_output(['pvs', '--noheadings', '--separator', r'|'], '|', 6):
```
I think you forgot to modify the separator as well, same in the other 2 calls.

Reply:
The pv is incorrectly parsed without that:
```
"pvs": [
        {
            "attr": "",
            "fmt": "03.0-part2|rhel_ibm-p8-kvm-03-guest-02|lvm2|a--|<39.00g|0",
            "pfree": "",
            "psize": "",
            "pv": "/dev/disk/by-path/pci-0000",
            "vg": "00"
        }
    ],

```

Reply:
Completly missed that, thanks!
Run it without to get the same message u got:
```
"pvs": [
        {
            "attr": "",
            "fmt": "03.0-part2|rhel_ibm-p8-kvm-03-guest-02|lvm2|a--|<39.00g|0",
            "pfree": "",
            "psize": "",
            "pv": "/dev/disk/by-path/pci-0000",
            "vg": "00"
        }
    ],
```
After fix the result should be now correct:
```
     "pvs": [
        {
            "attr": "a--",
            "fmt": "lvm2",
            "pfree": "0",
            "psize": "<39.00g",
            "pv": "/dev/disk/by-path/pci-0000:00:03.0-part2",
            "vg": "rhel_ibm-p8-kvm-03-guest-02"
        }
    ],
```
```
$ pvs -a
  PV                                       VG                          Fmt  Attr PSize   PFree
  /dev/disk/by-path/pci-0000:00:03.0-part1                                  ---       0     0
  /dev/disk/by-path/pci-0000:00:03.0-part2 rhel_ibm-p8-kvm-03-guest-02 lvm2 a--  <39.00g    0
```